gems/quilt_rails/lib/generators/quilt/demo_app/demo_app_generator.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Quilt
+  class DemoAppGenerator < Rails::Generators::Base
+    source_root File.expand_path('templates', __dir__)
+    desc "This generator adds a demo React app with all the quilt toolings."
+
+    def install_js_dependencies
+      say "Installing application js dependencies"
+      system("yarn add "\
+        "@shopify/react-router "\
+        "@shopify/polaris "\
+        "@shopify/react-async "\
+        "@shopify/react-network "\
+        "@shopify/react-performance "\
+        "@shopify/react-i18n "\
+        "@shopify/react-i18n-universal-provider") unless Rails.env.test?
+    end
+
+    def add_polaris_sk_config
+      file_path = "config/sewing-kit.config.ts"
+
+      unless File.exist?(file_path)
+        inject_into_file(
+          file_path,
+          File.read(File.expand_path(find_in_source_paths('path_import.ts'))),
+          after: "import {Plugins, Env} from '@shopify/sewing-kit';"
+        )
+
+        gsub_file(
+          file_path,
+          "plugins: [", "plugins: [#{File.read(File.expand_path(find_in_source_paths('polaris_sass_plugin.ts')))}"
```

Comment:
> As it stands you'll likely have to resort to pattern matching in some way to do this.

 @gmcgibbon for the pattern matching in both `gsub_file` and `inject_into_file`, can I use Regex in place of plain text?

Reply:
Yes. See thor's documentation: https://rubydoc.info/github/wycats/thor/master/Thor%2FActions:gsub_file